### PR TITLE
Fixed a test in Login.test.js

### LINKED ISSUE #10: Fix another test in Login.test.js
## Issue

Please troubleshoot the test on line 73 of Login.test.js. You can replace the .skip snippet at the beginning of the text declaration to .only, and run the test using `npm run test Login`.

Test should ensure that clicking on the "Sign Up" button makes the text of the same button change to "Sign In". I'm not sure why it isn't working! Please take a look and see if you can figure it out.

## Additional Resources

You may find it helpful to read the docs on [testing Apollo/GraphQL](https://www.apollographql.com/docs/react/development-testing/testing/).

### PR DESCRIPTION
**Pull request by guptabhaskar**
Fixes #10 
Assigned role="button" to StyledSpan for using getByRole in Login.test.js.

### CODE REVIEW

**Comment by capndave** [comment]
Thanks @guptabhaskar !

File: src/pages/Login/Login.test.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ describe("<Login /> view", () => {
     )
   })
 
-  it.skip("calls SIGNUP_MUTATION without error", async () => {
+  it.only("calls SIGNUP_MUTATION without error", async () => {
```
Comment (harpalassi) [inline src/pages/Login/Login.test.js:73]: remove only

Comment (guptabhaskar) [reply src/pages/Login/Login.test.js:73]: should I change it back to skip? I haven't worked with test cases before.

Comment (harpalassi) [reply src/pages/Login/Login.test.js:73]: it's all good, didn't realize dave had already merged. i'll take care of it.

Comment (guptabhaskar) [reply src/pages/Login/Login.test.js:73]: I fixed another issue in the same file and changed it back in that one.

Comment (guptabhaskar) [reply src/pages/Login/Login.test.js:73]: Thanks for reviewing @harpalassi 😊

Comment (guptabhaskar) [reply src/pages/Login/Login.test.js:73]: > should just be like `it("calls ...`
> `only` will make it so the other tests won't run anymore.
Oops, I added skip back 🤦‍♂️ in that PR. Will fix that before merging.

Comment (harpalassi) [reply src/pages/Login/Login.test.js:73]: nice, to clarify - you don't need to change it back to skip. just "it" by itself is what is needed here.

Comment (guptabhaskar) [reply src/pages/Login/Login.test.js:73]: Done!

**Pull request merged**